### PR TITLE
36 add info about rses and what we define as research

### PR DESCRIPTION
**Pull request by CaptainSifff**

### CODE REVIEW

**Comment by CaptainSifff** [comment]
@jpthiele , @jcohen02 I could use your help in elaborating on the definition of research discussion. Thanks!

**Comment by CaptainSifff** [comment]
Research Software includes source code files,
algorithms, scripts, computational workflows and
executables that were created during the
research process or for a research purpose.”, https://doi.org/10.5281/zenodo.5504016

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ machines, and they also need to be accessible, interoperable and reusable. More
 recently the FAIR principles have been extended specifically to research
 software [@FAIR4RS].
 
+In this paper we define Research Software to include source code files,
+algorithms, scripts, computational workflows and executables that were created
+during the research process or for a research purpose.
+This definition is broader than in [@FAIR4RS] and is the outcome of a recent 
+discussion in [@definingrs2021].
+
```
Comment (mschwarzmeier) [inline paper_teaching-learning-RSE.md:82]: What about documentation (and also tests)? As sort of an broader definition of "software+".
These are needed to understand and use (let alone further develop) the software.

File: paper_teaching-learning-RSE.md [hunk 2]
```diff
@@ -74,6 +74,12 @@ machines, and they also need to be accessible, interoperable and reusable. More
 recently the FAIR principles have been extended specifically to research
 software [@FAIR4RS].
 
+In this paper we define Research Software to include source code files,
+algorithms, scripts, computational workflows and executables that were created
+during the research process or for a research purpose.
```
Comment (mschwarzmeier) [inline paper_teaching-learning-RSE.md:79]: What about documentation (and also tests)? As sort of an broader definition of "software+".
These are needed to understand and use (let alone further develop) the software.

Comment (CaptainSifff) [reply paper_teaching-learning-RSE.md]: I gave it a definition section, that tries to cover that. maybe you can have a look again?

**Comment by CaptainSifff** [comment]
@jcohen02 , @mhagdorn, I think I'm through with this and we could merge this. I've started a section collecting some definitions(Probably due to @mschwarzmeier ...) and a section for collecting what is german specific and what not(maybe it shouldn't be a section). @jpthiele 's discussion of Wissenschaft and Forschug is in there, now.

File: paper_teaching-learning-RSE.md [hunk 3]
```diff
@@ -127,10 +127,35 @@ The skills themselves belong to one of the three main categories needed to perfo
...
+'Organizational Infrastructures' focuses on changes that are required to at best support
+the training of new RSEs.
 
 FIXME (finalization): Exchange chapter titles by numbers?
 
+
+## Some Definitions
+A couple of definitions are in order.
+First as software we define source code, documentation, tests
+and all other human-created artifacts during the development process
```
Comment (mhagdorn) [inline paper_teaching-learning-RSE.md:139]: artefact - BE spelling

File: paper_teaching-learning-RSE.md [hunk 4]
```diff
@@ -127,10 +127,35 @@ The skills themselves belong to one of the three main categories needed to perfo
...
+The german 'Wissenschaft' encompasses all domains of publicly funded
+research, while the english 'science' is generally limited to natural sciences.
+Therefore, we will use 'research' to encompass all domains therefore gives the employability of RSEs.
+Of course 'research' as well as 'Forschung' is not limited to public funding
+but also part of industrial and other private companies.
+We surmise that the same software engineering and team skills will be needed there,
+but we limit our considerations to the views of public research.
+
+This enables us to define Research Software in this paper to include source code files,
+algorithms, scripts, computational workflows and executables that were created
```
Comment (CaptainSifff) [inline paper_teaching-learning-RSE.md:154]: things that keep the ecosystem running?

Comment (CaptainSifff) [reply paper_teaching-learning-RSE.md:154]: Research Software **Engineering**

File: paper_teaching-learning-RSE.md [hunk 5]
```diff
@@ -127,10 +127,35 @@ The skills themselves belong to one of the three main categories needed to perfo
...
+during the research process or for a research purpose.
+This definition is broader than in [@FAIR4RS] and is the outcome of a recent
+discussion in [@definingrs2021].
+
```
Comment (hvwaldow) [inline paper_teaching-learning-RSE.md:158]: Under Research Software Engineering we also want to subsume all activities that are connected to maintain and develop the ecosystem in which research software runs. This includes system-administrator and Dev-Ops tasks that are specific for research.

File: paper_teaching-learning-RSE.md (hunk 4 above)
Comment (CaptainSifff) [reply paper_teaching-learning-RSE.md:154]: Engineering makes these things work together!

**Comment by CaptainSifff** [comment]
@mhagdorn @hvwaldow @jlinx @knarrff @mschwarzmeier @jngrad 
In the definition section, I now added a definition of the RSE. Established ones often focus exclusively on the software aspect.
I deliberately tried to use a flexible wording without using the words *maintenance* or *administration* and instead focuses on the creation of new things. This facilitates the employability in the german research landscape. If you agree with this I can merge it.
Best,

**Review by jngrad — APPROVED** [review]

**Pull request merged**